.travis.yml

```diff
@@ -1,11 +1,7 @@
 ---
-
-#
-# use container-based infrastructure
-# @see http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-# @see http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/
-#
-sudo: false
+sudo: required
+services:
+    - docker
```

Comment:
somehow these two things are required for `SO_REUSEPORT` to not blowup on travis :\